text/0227-separate-semantic-conventions.md

```diff
@@ -0,0 +1,155 @@
+
+- The semantic conventions will no longer be easily referencable form the specification.
+  - This is actually a benefit. We can ensure isolation of convention from
+    specification and require the Specification to use firm language for
+    attributes it requires, like `service.name`.
+  - We will provide x-links from existing location to the new location.
+- Semantic Convention version will no longer match the specification version.
+  - Instrumentation authors will need to consume a separate semantic-convention
+    bundle from Specification bundle. What used to be ONE upgrade effort will
+    now be split into two (hopefully smaller) efforts.
+  - We expect changes from Semantic Conventions and Specification to be
+    orthogonal, so this should not add significant wall-clock time.
+
+Initially this repository would have the following ownership:
+
+- Approvers
+  - [Liudmila Molkova](github.com/lmolkova)
+- Approvers (HTTP Only)
+  - [Trask Stalnaker](github.com/trask)
+- Maintainers
+  - [Josh Suereth](github.com/jsuereth)
+  - [Armin Reuch](github.com/arminru)
+  - [Reiley Yang](github.com/reyang)
+
+That is, Maintenance would initially continue to fall on (a subset of) the
+Technical committee. Approvers would start targeted at HTTP semantic convention
+stability and expand rapidly as we build momentum on semantic conventions.
+
+## Prior art and alternatives
+
+When we evaluate equivalent communities and efforts we see the following:
+
+- `OpenTracing` - had specification and [semantics](https://github.com/opentracing/specification/blob/master/semantic_conventions.md)
+  merged.
+- `OpenCensus` - had specification and semantics merged. However, OpenCensus
+  merged with OpenTelemetry prior to mass adoption or stable release of its
+  specification.
+- `Elastic Common Schema` - the schema is its own project / document.
+- `Prometheus` - Prometheus does not define rigid guidelines for telemetry, like
+  semantic conventions, instead relying on naming conventions and
+  standardization through mass adoption.
+
+## Open questions
+
+This OTEP doesn't address the full needs of tooling and codegen that will be
+needed for the community to shift to a separate semantic convention directory.
+This will require each SiG that uses autogenarated semantic conventions to
+adapt to the new location.
+
+## Future possibilities
+
+This OTEP paves way for the following desirable features:
+
+- Semantic Conventions can decide to bump major version numbers to accommodate
+  new signals or hard-to-resolve new domains without breaking the Specification
+  version number.
+- Semantic Conventions can have dedicated maintainers and approvers.
+- Semantic Conventions can restructure to better enable subject matter experts
+  (SMEs) to have approver/CODEOWNER status on relevant directories.
+
+There is a desire to move semantic conventions to domain-specific directories
+instead of signal-specific. This can occur after a separation of the repository.
```

Comment:
```suggestion
instead of signal-specific. This can occur after a separation of the repository
and will be proposed and discussed separately from this OTEP.
```